### PR TITLE
LPS-118086 Trim more features down

### PR DESCRIPTION
**Pull request by csierra**
Completely remove pepper support. Remove configuration and hardcode one
message digest implementation. Since no configuration is possible we can
get rid of:

* `CryptoHashVerificationContext`
* `CryptoHashProviderResponse`

Make salt mandatory since configuration is fixed.

### CODE REVIEW

File: modules/apps/portal-crypto-hash/portal-crypto-hash-impl/src/main/java/com/liferay/portal/crypto/hash/internal/CryptoHasherImpl.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/**
```
Comment (liferay-appsec) [inline modules/apps/portal-crypto-hash/portal-crypto-hash-impl/src/main/java/com/liferay/portal/crypto/hash/internal/CryptoHasherImpl.java:1]: When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation:

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by csierra** [comment]
ci:test:sf

**Comment by csierra** [comment]
ci:test:sf

**Comment by csierra** [comment]
ci:test:sf

**Comment by csierra** [comment]
sent to Brian directly here: https://github.com/brianchandotcom/liferay-portal/pull/97866

**Pull request closed without merging**